.github/workflows/ci.yml

```
@@ -16,16 +16,10 @@ jobs:
       - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)